core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/definitions/ldap/implementations/UnboundidLdapService.scala

```diff
@@ -336,6 +339,12 @@ object LdapConnectionConfig {
     final case class CustomUser(dn: Dn, password: PlainTextSecret) extends BindRequestUser
   }
 
+  sealed trait CircuitBreaker
+  object CircuitBreaker {
+    case object Disabled extends CircuitBreaker
+    final case class Enabled(maxFailures: Int, resetDuration: FiniteDuration Refined Positive) extends CircuitBreaker
```

Comment:
maxFailures also can be `Int Refined Positive`

_(We can do it in a second part of the task)_